components/admin_console/brand_image_setting.jsx

```diff
@@ -214,3 +203,45 @@ export default class BrandImageSetting extends React.PureComponent {
         );
     }
 }
+
+function UploadButton(props) {
```

Comment:
I am wondering if it would be better to deconstruct your props here so that it sort of self-documents.

    const UploadButton = ({primaryClass, status, disabled, onClick}) => {

Then you just remove `props.` from the calls in the function

Reply:
I'm 0/5 on that. Either way works fine.

Reply:
I just like the self documentation. I'm 1/5. Don't think it is crucial, but feel it is good practice.